src/main/java/nl/tudelft/contextproject/script/Script.java

```diff
@@ -152,7 +173,9 @@ public Shot getCurrentShot() {
         try {
             return shots.get(current);
         } catch (Exception e) {
-            return null;
+            Camera dummyCamera = new Camera();
+            dummyCamera.setNumber(-1);
+            return new Shot(-1, "-1", dummyCamera, new InstantPreset(new CameraSettings(), -1), "No shot");
```

Comment:
Great catch, that would indeed be a huge problem with cameras appearing out of nowhere.